packages/iis/manifest.yml

```
@@ -0,0 +1,51 @@
+name: iis
+title: IIS
+version: 0.1.0
+description: IIS Integrations
+type: integration
+icons:
+- src: /img/iis.svg
+  title: iis
+  size: 100x100
+  type: image/svg+xml
+format_version: 1.0.0
+license: basic
+categories: []
+release: beta
+conditions:
+  kibana.version: ^7.9.0
+screenshots:
+- src: /img/kibana-iis.png
+  title: kibana iis
+  size: 1960x2820
+  type: image/png
+- src: /img/metricbeat-iis-application-pool-overview.png
+  title: metricbeat iis application pool overview
```

Those tittles is suggested to be like:
```suggestion
  title: Metricbeat IIS Application Pool Overview
```